config/config-example.js

```diff
@@ -179,8 +179,12 @@ exports.customavatars = {
 // tourroom - specify a room to receive tournament announcements (defaults to
 // the room 'tournaments').
 // tourannouncements - announcements are only allowed in these rooms
+// tournamentdefaultplayercap - a set cap of how many players can be in a tournament
+// istournamentsrated - toggles tournaments being ladder rated (true) or not (false)
 exports.tourroom = '';
 exports.tourannouncements = [/* roomids */];
+exports.tournamentdefaultplayercap = false;
+exports.istournamentsrated = false;
```

Comment:
To make these consistent with the others, I think I'd prefer `tourdefaultplayercap` and `ratedtours`